### DISCUSSION TITLE
Multiple Pages Option?

### DISCUSSION THREAD

**Discussion by marzel1** [Lissy93/dashy#404] [Ideas]
Hello everyone and happy new year.

First of all a big THANK YOU to Lissy93 for the great work on Dashy! I switched from Homer to Dashy and so far everything I tried works fine. With all the docs, explanation and examples you created, it's pretty easy to find your way around. But one thing I haven't found yet or it doesn't exist yet on Dashy, multiple pages. If you have a lot of links like I do and also want to use some widgets, the page will quickly become fuller and fuller. It would be nice if you could switch to the different page with the navLinks and so organize the links and widgets better.

Does this possibility already exist and I have only seen it over or are there plans to introduce such a function?

Thanks Marzel

**Comment by lissy93** [comment]
Heya @marzel1 
Fully agree, we definatley need multi-page support, I think several others have suggested this too. I actually just posted a roadmap for next features (in #405) and mutli-page support is on the list :)

I'm not sure of a time frame, as this feature is dependant on changing how the config file is loaded in, but I will keep you updated on this thread, and let you know when completed.

**Comment by marzel1** [comment]
Thanks for your quick reply @Lissy93

A long roadmap, with many interesting points. I'm already looking forward to the next Update. :D 

Thanks again for your great work.
